makefile

```diff
@@ -14,10 +14,20 @@ build: clear install
 	@rm -rf dist
 	@NODE_ENV=production webpack --bail --color --config __config__/webpack/webpack.config.js
 
+build-ci: install
+	$(call log, "building production bundles")
+	@echo '' # just a spacer
+	@rm -rf dist
+	@CI=true NODE_ENV=production webpack --config __config__/webpack/webpack.config.js
+
 dev: clear install
 	$(call log, "starting DEV server...")
 	@NODE_ENV=development node __server__/development.js
 
+ci: validate
```

Comment:
task names should really describe what they do, not where they run :)

Reply:
Going to remove this because of simon's comments as well as yours and because I realised the build.sh script isn't there so it wouldn't work anyway